swan-lake/integration-tutorials/transforming-message-formats.md

```diff
@@ -279,7 +287,7 @@ Follow the instructions given in this section to develop the service.
 
     - Use the `is` check to check whether the response is `ReservationResponse` and return it as is (i.e., reservation successful).
 
-    - If the response is not a `ReservationResponse` record, log the information at `ERROR` level. Return a "NotFound" response if the response from the hospital service is an `http:ClientRequestError` response or an "InternalServerError" response otherwise.
+    - If the response is not a `ReservationResponse` record, log the information at `ERROR` level. Return a `http:NotFound` response if the response from the hospital service is an `http:ClientRequestError` response or an `http:InternalServerError` response otherwise.
```

Comment:
```suggestion
    - If the response is not a `ReservationResponse` record, log the information at `ERROR` level. Return an `http:NotFound` response if the response from the hospital service is an `http:ClientRequestError` response or an `http:InternalServerError` response otherwise.
```